docs/source/conf.py

```diff
@@ -65,6 +65,7 @@
     'nbsphinx',
     'sphinx.ext.autodoc',
     'sphinx.ext.doctest',
+    'sphinx.ext.imgconverter',
```

Comment:
This is needed to render some svg images in tutorials. This extension converts svg images to a format that pdflatex can work with.